backend/src/main/java/org/sefglobal/core/controller/MultiverseController.java

```diff
@@ -99,8 +129,30 @@ public Event createEvent(@Valid @RequestBody Event event) {
         return eventRepository.save(event);
     }
 
-    //todo: Implement an endpoint to update an event
-    //todo: Implement an endpoint to delete an event (change active status)
+    @PutMapping("/events/{id}")
+    public Event updateEvent(@PathVariable long id,
+            @Valid @RequestBody Event event) throws ResourceNotFoundException {
+        if (eventRepository.existsById(id)) {
+            return eventRepository.save(event);
+        } else {
+            String message = "Event does not exists for event id : " + id;
+            logger.error(message);
+            throw new ResourceNotFoundException(message);
+        }
+    }
+
+    @DeleteMapping("/events/{id}")
+    public Event deleteEvent(@PathVariable long id) throws ResourceNotFoundException {
+        if (eventRepository.existsById(id)) {
+            Event event = eventRepository.getOne(id);
+            event.setStatus("REMOVED");
```

Comment:
Add status values to a constants class and refer from there instead of hardcoding